source/operations/monitoring/grafana.rst

```diff
@@ -20,6 +19,13 @@ Prerequisites
 - An existing MinIO deployment with network access to the Prometheus deployment
 - `Grafana installed <https://grafana.com/grafana/download>`__
 
+.. admonition:: Grafana dashboards use metrics version 2
+   :class: note
+
+   The MinIO Grafana dashboards are designed for use with :ref:`metrics version 2 <minio-metrics-v2>`.
+   For more about metrics API versions, see :ref:`Metrics and alerts. <minio-metrics-and-alerts>`
```

Comment:
Should this include a statement along the lines of

```suggestion
   For more about metrics API versions, see :ref:`Metrics and alerts. <minio-metrics-and-alerts>`
   
   Version 3 metrics require creating your own dashboard.
```

Reply:
Oh yes, that would be excellent information to include. Also adding a link to the relevant Grafana docs.